src/seedu/addressbook/data/person/Person.java

```diff
@@ -9,6 +9,9 @@
  * Guarantees: details are present and not null, field values are validated.
  */
 public class Person implements ReadOnlyPerson {
+	
+	private static int nextSequenceNumber = 1;
```

Comment:
Notice that your indentation mechanism uses tabs for indentation. 
The coding standards we follow require using 4 spaces for indentation.
Tip: you can set Eclipse to do this for you, check out how!